src/dnsmasq/helper.c

```diff
@@ -128,10 +124,17 @@ int create_helper(int event_fd, int err_fd, uid_t uid, gid_t gid, long max_fd)
 	      /* return error */
 	      send_event(err_fd, EVENT_USER_ERR, errno, daemon->scriptuser);
 	    }
+	  /**** Pi-hole modification ****/
+	  log_info("Starting script helper FAILED");
```

Comment:
Is this a warning/error?

```suggestion
	  log_err("Starting script helper FAILED");
```